src/app/screens/DetailArticle/index.tsx

```diff
@@ -8,13 +8,16 @@ import icAddActive from '@assets/TabBar/icAddpostActive.png';
 import icFavouriteInactive from '@assets/TabBar/icFavoriteInactive.png';
 import icFavouriteActive from '@assets/TabBar/icFavoriteActive.png';
 import icDefaultArticleImage from '@assets/icons/icDefaultArticleImage.jpg';
+import icDelete from '@assets/icons/icDelete.png';
+import icEdit from '@assets/icons/icEdit.png';
+import Routes from '@constants/routes';
 
 import styles from './styles';
 import testIds from './testIds';
 
 interface Props extends Navigation {}
 
-function DetailArticle({ route }: Props) {
+function DetailArticle({ route, navigation }: Props) {
```

Comment:
what do you think about this :

 interface Props {
  route: {
    params: {
      article: Article;
    };
  };
}
and navigation from: import useNavigation from '@components/AppNavigator/helper'; 
if you don't like ignore this